app/server/app/models/index.js

```diff
@@ -0,0 +1,85 @@
+const fs = require("fs");
+const path = require("path");
+const Sequelize = require("sequelize");
+const basename = path.basename(__filename);
+const db = {};
+const logger = require("../utilities/logger.js");
+const log = logger.logger;
+require("dotenv").config();
+
+let isLocal = false;
+let isDevelopment = false;
+let isStaging = false;
+
+let database_host = "";
+let database_port = "";
+const database_name = process.env.DB_NAME ?? "expert_query";
+const database_user = process.env.DB_USERNAME;
+const database_pwd = process.env.DB_PASSWORD;
```

Comment:
Minor note, but two different naming conventions are used here with snake_case and camelCase. Is this to make it clear they're environment variables?

Reply:
I'll switch to camelCase for consistency and change the name to more closely match the environment variable names (i.e., `database_name` will be changed to `dbName`).